third_party/move/move-compiler-v2/tests/checking/typing/index_err.exp

```diff
@@ -0,0 +1,25 @@
+
+Diagnostics:
+warning: unused alias
+  ┌─ tests/checking/typing/index_err.move:9:19
+  │
+9 │         use 0x42::test;
+  │                   ^^^^ Unused 'use' of alias 'test'. Consider removing it
```

Comment:
This error is not ideal, since the code clearly has a use of test.  Perhaps you can record the use of `test::R` below so this error is suppressed.  I can't quite find it at the moment.

Reply:
Updated the code to get rid of this error message